src/platforms/esp32/test/main/test_erl_sources/test_crypto.erl

```diff
@@ -175,7 +177,8 @@ test_crypto_one_time() ->
 
     ok.
 
-test_available_ciphers() ->
+test_available_ciphers(Model) when Model =:= esp32 ->
```

Comment:
can't be just `test_available_ciphers(esp32)` ?

Reply:
TIL - I'm elixir dev, so minimal Erlang skills.